### PR TITLE
dependabot, do not rebase merge

### PR DESCRIPTION
**Pull request by talgie-ll**
The dependabot-auto-merge workflow was implemented to use rebase-merging, however some repos do not have that enabled. This update improves compatibility across the projects where we wish to use this action.

### CODE REVIEW

**Review by FrankKerschbaumer3 — APPROVED** [review]

File: dependabot-auto-merge/action.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ runs:
       if: ${{ steps.auto-merge.conclusion == 'success' }}
       run: |
         gh pr review --approve "$PR_URL"
-        gh pr merge --auto --rebase "$PR_URL"
```
[the remark this answers is not in the document]
Comment (FrankKerschbaumer3) [reply dependabot-auto-merge/action.yml:62]: we can turn that on if it wont have any unintended consequences

[the remark this answers is not in the document]
Comment (timlo) [reply dependabot-auto-merge/action.yml:62]: Does the command fail so an OR can work? 

```
gh pr merge --auto --rebase "$PR_URL"  || gh prmerge --auto --merge "$PR_URL"
```

[the remark this answers is not in the document]
Comment (jroman00) [reply dependabot-auto-merge/action.yml:62]: Worth testing it out

**Comment by donofriov** [comment]
CleanShot 2022-10-14 at 15 29 50@2x

**Pull request merged**